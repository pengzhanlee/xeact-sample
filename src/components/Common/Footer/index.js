import React, {Component} from "react";


export default
class Footer extends Component {

    render() {
        return (
            <div>
                <footer className="main-footer clearfix">
                    <div className="pull-right hidden-xs">
                        <strong>Copyright © 2017</strong>  成都研究院-网站业务部-运营支持部
                    </div>
                </footer>
                <div className="control-sidebar-bg" />
            </div>
        );
    }

}
