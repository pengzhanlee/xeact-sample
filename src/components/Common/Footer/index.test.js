// Link.react-test.js
import React from 'react';
import Footer from './';
import renderer from 'react-test-renderer';

test('rendered right', () => {
    const component = renderer.create(
        <Footer />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});