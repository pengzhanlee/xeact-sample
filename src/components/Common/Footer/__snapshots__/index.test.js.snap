// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendered right 1`] = `
<div>
  <footer
    className="main-footer clearfix"
  >
    <div
      className="pull-right hidden-xs"
    >
      <strong>
        Copyright © 2017
      </strong>
        成都研究院-网站业务部-运营支持部
    </div>
  </footer>
  <div
    className="control-sidebar-bg"
  />
</div>
`;
